papermill/cli.py

```diff
@@ -129,6 +133,22 @@ def papermill(
     output in the destination notebook.
 
     """
+    # Check if input is being piped
+    if (not notebook_path) or (not output_path) and S_ISFIFO(os.fstat(0).st_mode):
```

Comment:
```suggestion
if (not notebook_path or not output_path) and S_ISFIFO(os.fstat(0).st_mode)
```
of the logic won't work quite correctly.

Reply:
Probably means we need a unittest for this check too

Reply:
Interesting! Apparently I misunderstood what my editor told me... (anyway, as discussed this part goes away with the next commit)

![image](https://user-images.githubusercontent.com/29915202/58853329-309a1280-869a-11e9-801e-d324c1fd3d32.png)